tests/integration/security/sds_ingress_k8sca/quic/ingress_test.go

```diff
@@ -0,0 +1,98 @@
+// +build integ
+//  Copyright Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package quic
```

Comment:
IMO we don't need this one. `k8sca` is a variant on `sds_ingress`. We don't need all combinatoric explosion of possible options - `k8sca` and `quic` have no relation and are extremely unlikely to conflict

Reply:
Removed this. Looking at it, I guess only the PKI stuff changes. Correct? As long as we have the right CA certs for verification, it does not affect QUIC-TLS handshake.